doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -663,6 +633,129 @@ Reserved keywords
    section are examples of user-defined variables (e.g. ``inCollection``) and
    are not required by BPS.
 
+.. _managing-job-memory:
+
+Managing job memory requirements
+--------------------------------
+
+The primary parameters controlling job memory requirements is called
+``requestMemory``.  It specifies the amount of memory (in MiB) each job will
+have at its disposal during the execution of the workflow. Its default value
+is 2048 (2 GiB).
+
+The default value can be overridden globally for all the jobs as well as for
+jobs running specific tasks.  For example, including the lines below in your
+BPS config will result in all jobs having 4 GiB of memory during their
+execution *except* jobs running ``skyCorr`` task that will be able to use up to
+8 GiB of RAM as task specific setting takes precedence over the global
+one if both are present in your BPS config.
+
```

Comment:
Due to Brian's question the other day in slack, wondering if we need information here about memory and clustering.  It is there in the user-defined dimension clustering (end first paragraph) so maybe a pointer?

Reply:
I'll add a sentence pointing to the example in the section about clustering.